react/__mocks__/vtex.place-components.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { AddressContext } from 'vtex.address-context'
+
+const { useAddressContext } = AddressContext
+
+export const PlaceDetails: React.FC = () => {
+  const { address } = useAddressContext()
+
+  return <div>{JSON.stringify(address, null, 2)}</div>
```

Comment:
Is this ok? All that I think we should care about is that PlaceDetails renders the address details somehow. Or should we just remove this at all and go with an integration test?

Reply:
I believe this is fine, we could keep it this way until we can run unit tests and pull the implementation from other repositories